packages/main/pyproject.toml

```diff
@@ -36,7 +36,8 @@ packages = [
 python = "^3.6.2"
 docutils = "*"
 dataclasses = { version = "^0.7", python=">=3.6,<3.7" }
-rpaframework-core = "^6.4.0"
+# rpaframework-core = "^6.5.0"
+rpaframework-core = { path = "../core", develop = true}
```

Comment:
Temporary change until `6.5.0` of **rpaframework-core** will be available, so the CI passes entirely.